Update README.md

kkarunakaran89 opened the pull request:
1) Adding meeting timing and cadence
2) Adding Zoom link 
3) Adding link to notes and videos

Signed-off-by: kkarunakaran89 <109688962+kkarunakaran89@users.noreply.github.com>

SteveLasker reviewed (commented):
Thanks @kkarunakaran89 
I was just doing something similar for the go-cose community page, where I added a line for the ics: https://github.com/veraison/go-cose/pull/127/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R26

The ics for the scitt-community call would be: https://zoom.us/meeting/tZIucuisqj4sGNYoptUH4YDWxty7rLcEz5eH/ics

kkarunakaran89 commented:
> Thanks @kkarunakaran89 I was just doing something similar for the go-cose community page, where I added a line for the ics: https://github.com/veraison/go-cose/pull/127/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R26
> 
> The ics for the scitt-community call would be: https://zoom.us/meeting/tZIucuisqj4sGNYoptUH4YDWxty7rLcEz5eH/ics

Hi Steve, great catch! I have updated the link

yogeshbdeshpande commented on profile/README.md:
```diff
@@ -10,8 +10,9 @@ To work as a community to facilitate adoption of IETF Supply Chain Integrity, Tr
-- Link to meeting agenda, prior meeting notes and meeting videos will be made available
++ SCITT Community Meetings: Bi-Weekly on Wednesday 8AM PST/4PM UTC
++ SCITT Community Bi-Weekly Meeting (iCal link): [Click Here](https://zoom.us/meeting/tZIucuisqj4sGNYoptUH4YDWxty7rLcEz5eH/ics)
++ Meeting  Notes and Videos: [See link to meeting notes and previously recorded sessions](https://docs.google.com/document/d/1OBfJmZdxXfJ7MWHKmCaAn0Grn1LHxFOyFtPdH_U_dOQ/edit#)
```
Thanks for the PR
Minor point: Do we want to give edit permission to every one? 
The link points to edit# hence asking.

Rest all changes LGTM!

SteveLasker replied:
It's the default mode, and doesn't change permissions, but it's a good point to cleanup. Doing now...

SteveLasker reviewed (approved):
LGTM with minor point to cleanup the default edit parameter in the hackmd link
